tests/pom.xml

```diff
@@ -243,15 +277,31 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
         </property>
       </activation>
       <properties>
-        <deviceregistry.image>hono-service-device-registry-mongodb</deviceregistry.image>
+        <hono.deviceregistry.image>hono-service-device-registry-mongodb</hono.deviceregistry.image>
+        <hono.deviceregistry.resources.folder>deviceregistry-mongodb</hono.deviceregistry.resources.folder>
         <hono.mongodb.disabled>false</hono.mongodb.disabled>
         <hono.mongodb.host>hono-mongodb.hono</hono.mongodb.host>
         <hono.mongodb.port>27017</hono.mongodb.port>
         <hono.mongodb.username>device-registry@HONO</hono.mongodb.username>
         <hono.mongodb.password>device-registry-secret</hono.mongodb.password>
         <hono.mongodb.database.name>hono-it-tests</hono.mongodb.database.name>
-        <hono.deviceregistry.resources.folder>deviceregistry-mongodb</hono.deviceregistry.resources.folder>
-        <deviceregistry.supportsSearchDevices>true</deviceregistry.supportsSearchDevices>
+      </properties>
+    </profile>
+    <profile>
+      <id>protocol-adapters-quarkus</id>
```

Comment:
Maybe keep profile name a bit more generic as I can see we use Quarkus for other services not just protocol adapters? Or you want to configure that with separate profiles?

Reply:
No, I do not think that we need to use separate profiles for it. How about changing it to a more generic name once we start migrating the first non-adapter component?